### PR TITLE
1479/systems without declarations default

### LINKED ISSUE #1479: Config Wizard - Add systems without privacy declarations
As a user, when I complete a scan, I want to have access to the results of the scan so that I can add privacy information on my own schedule without losing the results of the scan.

**Details**
Today, when you perform an AWS scan, you need to add privacy declarations for each system and if you cancel, the systems from the scan aren't persisted, so you need to re-scan to add in the difference. 

**AC**
- Given that I have completed a scan in the config wizard, when I choose to add selected systems to my map, then they are created with empty privacy declarations.

### PR DESCRIPTION
**Pull request by LKCSmith**
Closes #1479 

### Code Changes

* [x] Remove system declaration step from config wizard if you're coming from a scanner and then re-route to datamap if using the scanner
* [x] **Plus users re-rout to datamap page, non-plus users reroute to systems page**
* [x] Manual systems entry follows the privacy declaration steps to set-up

### Steps to Confirm

* [ ] _list any manual steps taken to confirm the changes_

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation Updated:
  * [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  * [ ] documentation issue created (tag docs-team to complete issue separately)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`

### Description Of Changes

_Write some things here about the changes and any potential caveats_

### CODE REVIEW

File: clients/admin-ui/cypress/e2e/config-wizard.cy.ts [hunk 1]
```diff
@@ -75,24 +75,8 @@ describe.skip("Config Wizard", () => {
...
-        .click();
-      cy.getByTestId("input-data_subjects").type(`user{enter}`);
-      cy.getByTestId("add-btn").click();
-      cy.getByTestId("next-btn").click();
-      cy.getByTestId("confirm-btn").click();
-
-      // Just the first system is reviewed.
-      cy.get(`[data-testid="system-reviewed"]`).should("have.length", 1);
-      cy.get(`[data-testid="system-needs-review"]`).should("have.length", 2);
+      // Non-manual entry skips declaration step
```
Comment (LKCSmith) [inline clients/admin-ui/cypress/e2e/config-wizard.cy.ts:78]: Not to self: need to remove

File: clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx [hunk 2]
```diff
@@ -58,15 +63,32 @@ const ScanResultsForm = () => {
...
-    } else {
+    }
+
+    if (!features.plus && systems.length <= values.selectedKeys.length) {
       dispatch(chooseSystemsForReview(values.selectedKeys));
       dispatch(changeStep());
     }
+
+    if (features.plus && systems.length <= values.selectedKeys.length) {
+      // NOTE: I don't know if this will re-route correctly yet in a live environment?
```
Comment (LKCSmith) [inline clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx:74]: Note to self: need to review prior to review

File: clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx [hunk 3]
```diff
@@ -58,15 +63,32 @@ const ScanResultsForm = () => {
...
+      router.push(`/datamap`);
+    }
   };
 
   const confirmRegisterSelectedSystems = () => {
-    dispatch(chooseSystemsForReview(selectedKeyValues));
-    dispatch(changeStep());
+    // Manual system setup
+    // go to privacy declarations step
+    if (!features.plus) {
```
Comment (chriscalhoun1974) [inline clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx:82]: Should the multiple if statements be combined to an if/else?

Comment (chriscalhoun1974) [inline clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx:82]: Should the multiple if statements be combined to an if/else statement?

**Review by chriscalhoun1974 — APPROVED** [review]
Left very minor suggestions.

**Pull request merged**